cross-chain/sei/hardhat.config.ts

```diff
@@ -0,0 +1,143 @@
+import { HardhatUserConfig } from "hardhat/config"
+import "@keep-network/hardhat-helpers"
+import "@nomiclabs/hardhat-ethers"
+import "@nomiclabs/hardhat-etherscan"
+import "@nomiclabs/hardhat-waffle"
+import "@openzeppelin/hardhat-upgrades"
+import "@typechain/hardhat"
+import "hardhat-contract-sizer"
+import "hardhat-dependency-compiler"
+import "hardhat-deploy"
+import "hardhat-gas-reporter"
+import "dotenv/config"
+
+const config: HardhatUserConfig = {
+  solidity: {
+    compilers: [
+      {
+        version: "0.8.17",
+        settings: {
+          optimizer: {
+            enabled: true,
+            runs: 1000,
+          },
+        },
+      },
+      {
+        version: "0.8.15",
+        settings: {
+          optimizer: {
+            enabled: true,
+            runs: 1000,
+          },
+        },
+      },
+    ],
+  },
+
+  paths: {
+    artifacts: "./build",
+  },
+
+  // Compile external dependencies so their artifacts are available
+  dependencyCompiler: {
+    paths: [
+      "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol",
+      "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol",
+    ],
+  },
+
+  // Hardhat-deploy configuration
+  namedAccounts: {
+    deployer: {
+      default: 0,
+    },
+    governance: {
+      default: 1,
+      seiTestnet: 0,
+      seiMainnet: "0x0000000000000000000000000000000000000000", // TBD
```

Comment:
Do we need to update it before deployment/merging?

Reply:
I will put the Tokenholder Governor for TBTC, is that fine? 

https://etherscan.io/address/0xd101f2b25bcbf992bdf55db67c104fe7646f5447#code